server/game/Card.js

```diff
@@ -57,6 +59,28 @@ class Card extends EffectSource {
             effect: AbilityDsl.effects.cardCannot('play', () => !this.game.firstThingThisTurn())
         });
 
+        // enraged
+        this.persistentEffect({
+            location: 'any',
+            printedAbility: false,
+            condition: () => {
+                return this.tokens.enrage >= 1;
+            },
+            match: this,
+            effect: AbilityDsl.effects.mustFightIfAble()
+        });
+
+        this.reaction({
```

Comment:
This probably needs to happen in the engine itself, rather than being a reaction